src/SFA.DAS.Reservations.Functions.LegalEntities.AcceptanceTests/Steps/LegalEntityUpdatedSteps.cs

```diff
@@ -68,15 +68,15 @@ public void ThenTheLegalEntityStateShouldBeSigned()
             Assert.IsTrue(legalEntity.AgreementSigned);
         }
 
-        [Then(@"the legal entity should be marked as a levy entity")]
+        [Then(@"the account should be marked as a levy entity")]
         public void ThenTheLegalEntityShouldBeMarkedAsALevyEntity()
         {
             var dbContext = Services.GetService<ReservationsDataContext>();
-            var legalEntity = dbContext.AccountLegalEntities.SingleOrDefault(ale =>
-                ale.AccountLegalEntityId.Equals(TestData.AccountLegalEntity.AccountLegalEntityId));
+            var account = dbContext.Accounts.SingleOrDefault(ale =>
+                ale.Id.Equals(TestData.AccountLegalEntity.AccountId));
 
-            Assert.IsNotNull(legalEntity);
-            Assert.IsTrue(legalEntity.IsLevy);
+            Assert.IsNotNull(account);
+            Assert.IsTrue(account.IsLevy);
 
         }
```

Comment:
I'd move this to account steps, also it's not used in any features currently. altho you have removed it from the ale feature.

Reply:
Created a new `Account.feature` file to cover this test, and also any other events received for the account being updated and created.